recipe/patches/0001-Add-llvm-wrap.patch

```diff
@@ -0,0 +1,60 @@
+From d32e019ee0e9ed53f1a631f2bdc2800bcd8239c0 Mon Sep 17 00:00:00 2001
+From: Jarrett Johnson <jarrett.johnson@schrodinger.com>
+Date: Sun, 23 Mar 2025 13:02:14 -0400
+Subject: [PATCH 1/2] Add llvm wrap
+
+---
+ subprojects/llvm/meson.build | 32 ++++++++++++++++++++++++++++++++
+ 1 file changed, 32 insertions(+)
+ create mode 100644 subprojects/llvm/meson.build
+
+diff --git a/subprojects/llvm/meson.build b/subprojects/llvm/meson.build
+new file mode 100644
+index 00000000000..bdb863a332e
+--- /dev/null
++++ b/subprojects/llvm/meson.build
+@@ -0,0 +1,34 @@
++project('llvm', ['cpp'])
++
++cpp = meson.get_compiler('cpp')
++
++_deps = []
++_prefix = get_option('prefix')
++_search = join_paths(_prefix, 'lib')
++ message(_search)
++
++# Find all LLVM libs via llvm-config
++_llvm_libs = run_command('llvm-config', '--libnames').stdout().strip().split()
```

Comment:
I honestly don't keep a windows machine handy, but is the problem something with the output of `llvm-config`? 

for example, there is no standard way to get the `zstd.dll.lib`

Sometimes:

* `zstd.dll` shared library
* `zstd.lib` fully static library
* `zstd.dll.lib` shared library used for linking. The actual functions are defined in `zstd.dll`.

If there is something that the `llvm` feedstock can do to help, we can raise an issue there.

Reply:
Right, the fact that `llvm-config` informs meson to look for `zstd.dll.lib` is the issue. Mesa suggests using another `meson.build` file on Windows as I did here (ref: https://docs.mesa3d.org/meson.html#llvm).

I have no idea if there's something that the `llvm-feedstock` can do. Probably a little above my head.

Reply:
do you have a windows machine handy?

The
```
llvm-config
```
command should report the correct libraries on windows. conda-forge compiles its own version of llvm so we should be able to "fix" this for long term usage.

Reply:
Right, this (plus the meson instructions) is what lead me to get around this issue.

```
llvm-config.exe --system-libs
```
reports `zstd.dll.lib` as one of the deps which is why I can't build with Meson taking its list of libraries from `llvm-config`.

Reply:
![image](https://github.com/user-attachments/assets/e90fbe14-de57-4fba-9698-ab713ed38603)

Reply:
Perfect, thanks!
https://github.com/conda-forge/llvmdev-feedstock/issues/319